projects/kit/components/slider/slider.style.less

```diff
@@ -10,7 +10,7 @@
 @ticks-thickness: 0.25rem;
 
 @track-color: var(--tui-slider-track-color);
-@thumb-color: currentColor;
+@thumb-color: var(--tui-slider-thumb-color);
```

Comment:
```ts
tuiSliderOptionsProvider({thumbColor: ''}
```

causes the following result:
![Screenshot 2023-08-29 at 14 41 33](https://github.com/taiga-family/taiga-ui/assets/35179038/3b760f79-4748-49ae-b335-7e9a00823b6b)

Reply:
fixed